test/unit/async-init.ts

```diff
@@ -15,20 +15,21 @@
  *  PERFORMANCE OF THIS SOFTWARE.
  */
 
-import '../'
+import '..'
 import { describe, it } from 'mocha'
 import AsyncInit from '../../src/utils/async-init'
 
 describe('AsyncInit', function () {
   it('composes Stamps', async () => {
     return AsyncInit.compose({
-      async init (val) {
-        return Promise.resolve(val)
+      async init (options: unknown) {
+        return await Promise.resolve(options)
```

Comment:
```suggestion
        return options
```
actually, `Promise.resolve` is not needed because `init` is async function